README.md

```diff
@@ -3,21 +3,132 @@
 ## Deployment Prerequisites
 
 ### Software
+
 In order to build and deploy MLSpace to your AWS account you will need the following software installed on your machine:
+
 - [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
 - [awscli](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
 - [nodejs](https://nodejs.org/en/download/) 18+
 - [docker](https://docker.com) or a compatible runtime for generating lambda layers
 - cdk (`npm install -g cdk`)
 
-
 ### Additional Information
+
 In addition to the required software you will also need to have the following information:
+
 - AWS account Id and region you'll be deploying MLSpace into (you'll need admin credentials or similar)
 - IdP information including the OIDC endpoint and client name
 
-### Configure deployment parameters
-Update the values in `lib/constants.ts` based on your specific deployment needs. Some of these will directly impact whether new resources are created within your account or whether existing resources (VPC, KMS, Roles, etc) will be leveraged.
+## Configuring MLSpace
+
+There are two options for configuring MLSpace for deployment:
+
+### Option 1 - configure using the MLSpace Config Wizard
```

Comment:
Should we mention this as the preferred option? Not blocking

Would like this capitalized: "Configure using..."

Reply:
Good idea, I'll mark Option 1 as "Recommended".

I'll also capitalize Configure